h2/src/test/scala/org/scalarelational/h2/SessionSpec.scala

```diff
@@ -78,9 +76,8 @@ class SessionSpec extends WordSpec with Matchers {
       s1.disposed should equal(false)
       s2.disposed should equal(false)
       s1 should be theSameInstanceAs s2
-      Time.waitFor(1.0, errorOnTimeout = true) {
-        s1.disposed
-      }
+      Thread.sleep(1000L)
```

Comment:
Unfortunately ScalaTest 3.0 hasn't been released yet, so I don't think we should be adding pre-release dependencies to our project if we can help it...even for testing.